streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -2116,4 +2132,37 @@ boolean needsInitializationOrRestoration() {
     void addTask(final Task task) {
         tasks.addTask(task);
     }
+
+    private boolean canTryLock(final TaskId taskId, final long nowMs) {
```

Comment:
Sorry, I forgot to add this comment before in my review. Could you please rename this method to `canTryInitializeTask()`? I think that makes more sense.

Reply:
@cadonna makes sense!